parameters/cookie_parameters.go

```diff
@@ -5,14 +5,17 @@ package parameters
 
 import (
 	"fmt"
-	"github.com/pb33f/libopenapi-validator/errors"
-	"github.com/pb33f/libopenapi-validator/helpers"
-	"github.com/pb33f/libopenapi/datamodel/high/base"
-	v3 "github.com/pb33f/libopenapi/datamodel/high/v3"
 	"net/http"
 	"strconv"
 	"strings"
+
+	"github.com/pb33f/libopenapi/datamodel/high/base"
+
+	"github.com/pb33f/libopenapi-validator/errors"
+	"github.com/pb33f/libopenapi-validator/helpers"
 	"github.com/pb33f/libopenapi-validator/paths"
+
+	v3 "github.com/pb33f/libopenapi/datamodel/high/v3"
```

Comment:
You used alias in gci, but I find the result strange here

```
         "github.com/pb33f/libopenapi/

         "github.com/pb33f/libopenapi-validator/

         v3 "github.com/pb33f/libopenapi
```


Maybe one of the [solutions](https://github.com/pb33f/libopenapi-validator/pull/101#discussion_r1832532789) I'm suggesting would provide something less strange